src/pages/admin/donate/components/generic-details/GenericDetails.tsx

```
@@ -71,32 +77,36 @@ export function GenericDetails<T extends { id: number } & FieldValues>({
         onAddFormVisibilityChange?.(false);
     };
 
     const handleSubmit = useCallback(
         async (data: T) => {
             if (onSubmit) {
                 await onSubmit(data);
-                setIsAddFormVisible(false);
+            } else if (isChildForm) {
+                onChangeItems?.((prevItems) => [...prevItems, data]);
                 onAddFormVisibilityChange?.(false);
             } else if (isChildForm) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)